finetune/base_models/glove/featurizer.py

```diff
@@ -0,0 +1,41 @@
+import tensorflow as tf
+
+from finetune.util.shapes import shape_list, lengths_from_eos_idx
+
+
+def embed(X, we):
+    e = tf.gather(we, X)
+    h = tf.reduce_sum(e, 2)
```

Comment:
I think this reduce sum is squashing out the "position" embeddings?? Might be worth slicing these off before the gather?